products/rancher-shell.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Rancher Shell
+addedAt: 2025-10-01
+category: server-app
+tags: suse rancher
+iconSlug: rancher
+permalink: /rancher-shell
+releasePolicyLink: https://www.suse.com/lifecycle#rancher
```

Comment:
There is nothing on https://www.suse.com/lifecycle#rancher about rancher shell : should this link be removed ?